x-pack/test/cases_api_integration/security_and_spaces/tests/common/index.ts

```diff
@@ -8,7 +8,7 @@
 import { FtrProviderContext } from '../../../common/ftr_provider_context';
 
 // eslint-disable-next-line import/no-default-export
-export default ({ loadTestFile }: FtrProviderContext): void => {
+export default ({ loadTestFile, getService }: FtrProviderContext): void => {
```

Comment:
I can't spot where the `getService` is used in this file. Maybe a left-over from some refactoring?